daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -123,6 +129,14 @@ cmdInspect =
     cmd = execInspect <$> inputFileOpt <*> outputFileOpt <*> jsonOpt
 
 
+cmdPackageNew :: Int -> Mod CommandFields Command
+cmdPackageNew numProcessors =
+    command "package-new" $
+    info (helper <*> cmd) $
+    progDesc "Compile the DAML project into a DAML ARchive (DAR)" <> fullDesc
```

Comment:
The capitalization here looks too much like a typo for my taste but your choice.

Reply:
fixed it also for the old package command.